testing/test-vectors/prng-tests.lisp

```diff
@@ -2,5 +2,8 @@
 (in-package :crypto-tests)
 
 (rtest:deftest :prng-fortuna (run-test-vector-file :prng *prng-tests*) t)
+(rtest:deftest :prng-fortuna-urandom (let ((prng (crypto:make-prng :fortuna :seed urandom)))
```

Comment:
Looks like the tests exit successfully even with an error: https://travis-ci.org/froydnj/ironclad/jobs/224327258

I've fixed the `URANDOM` bug, but someone who knows more about Travis CI & RT needs to figure out how to exit non-zero on errors …